<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1119</width>
    <height>889</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="bgwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="maximumSize">
    <size>
     <width>1121</width>
     <height>891</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#bgwidget{
background-color: qlineargradient(spread:pad, x1:0, y1:0.961, x2:1, y2:0, stop:0 rgba(33, 137, 205, 255), stop:1 rgba(255, 255, 255, 255))};</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>150</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 36pt &quot;MS Shell Dlg 2&quot;; color : rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>220</y>
      <width>351</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;; color : rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Sign in to your existing account</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Username">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>350</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_box">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>450</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>320</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
color:rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>420</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;; color:rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>150</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 36pt &quot;MS Shell Dlg 2&quot;; color : rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>Account</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>630</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton{
background-color:rgb(0, 0, 0);
color:rgba(255,255,255,210);
border-radius:25px;
font: 20pt &quot;MS Shell Dlg 2&quot;; color:rgb(255, 255, 255);
}

QPushButton#pushButton:hover{
background-color:qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(150, 123, 111, 219), stop:1 rgba(85, 81, 84, 226));
}

QPushButton#pushButton:pressed{
padding-left:5px;
padding-top:5px;
background-color:rgba(150, 123, 111, 255);
};</string>
    </property>
    <property name="text">
     <string>Log in</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Role">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>520</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgba(255,255,255,210);
border-radius:25px;
font: 12pt &quot;MS Shell Dlg 2&quot;; color:rgb(0, 0, 0);</string>
    </property>
    <item>
     <property name="text">
      <string>Pelanggan</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Teknisi</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Admin</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>570</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;
color:rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>Don't have an account ?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="create_account">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>560</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#create_account{
background-color:rgb(0, 0, 0);
color:rgba(255,255,255,210);
border-radius:14px;
font: 11pt &quot;MS Shell Dlg 2&quot;; color:rgb(255, 255, 255);
}

QPushButton#create_account:hover{
background-color:qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(150, 123, 111, 219), stop:1 rgba(85, 81, 84, 226));
}

QPushButton#create_account:pressed{
padding-left:5px;
padding-top:5px;
background-color:rgba(150, 123, 111, 255);
};</string>
    </property>
    <property name="text">
     <string>Create account</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
